clang-tools-extra/clang-tidy/misc/CMakeLists.txt

```diff
@@ -51,6 +52,7 @@ add_clang_library(clangTidyMiscModule
   genconfusable
   ClangDriverOptions
   )
+set_target_properties(clangTidyMiscModule PROPERTIES FOLDER "Clang Tools Extra/Libraries")
```

Comment:
why this is set for Misc module, but not for other check modules.

Reply:
This `set_target_properties` is redundant here, the same folder is already set by `add_clang_library` in #89741. This must be a leftover from cleaning up when I changed `add_clang_library` to set the folder instead of each of them individually. Thanks for noticing.